src/lambdas/lambda_delete_file.py

```diff
@@ -0,0 +1,45 @@
+from typing import Dict
+from .common import *
+import base64
+
+def base64url_decode(input):
+    return base64.urlsafe_b64decode(input + '==')
+
+
+def manage_token(auth_header: str) -> str:
+    parts = auth_header.split()
+    jwt = parts[1]
+
+    jwt_body_str: str = jwt.split('.')[1]
+    jwt_body: Dict = json.loads(base64url_decode(jwt_body_str))
+    username = jwt_body['username']
+
+    return username
+
+
+def lambda_delete_file(event: Dict, context):
+    body: Dict = json.loads(event['body'])
+    headers: Dict = event['headers']
```

Comment:
Pretty sure we can just use `dict` instead of importing `Dict` as a type annotation wherever necessary. From Python 3.9 especially `Dict` is obsolete, since `dict` can be made generic as well.
`Dict` is imported elsewhere as well.

Reply:
[dc0ad7c](https://github.com/magley/klau-drive/pull/24/commits/dc0ad7c068d2ef1df7c03a1c577ad0ef1c641dbd)